src/naksu/mebroutines/install/install.go

```diff
@@ -15,15 +15,28 @@ import (
 )
 
 // GetServer downloads vagrantfile and starts server
+// nolint: gocyclo 
```

Comment:
Tämähän on aika kompleksinen funktio (toki paljolti virheenkäsittelyn vuoksi), joten ihan ymmärrettävää että lintteri siitä varoittelee. Voihan tämän disabloida, mutta vaihtoehtoinen lähestymistapa on yksinkertaistaa funktiota esim. pilkkomalla sitä apufunktioihin, mikä voisi tehdä siitä helppolukuisemman.

Reply:
Päädyin nyt tekemään näin, vaikka minusta on höperöä tehdä asioita vain sen vuoksi, että joitain rakenteita funktiossa on 11 sijaan 10. Oikea ratkaisu olisi paketoida tämä virtuaalikoneen hallinta omaksi paketikseen, mutta nyt ei ole sen aika.